test/test_evaluation/test_evaluation.py

```diff
@@ -380,6 +380,10 @@ def test_silent_exception_in_target_function(self):
                 """'save_targets_ensemble'",)""",
                 """AttributeError("'BackendMock' object has no attribute """
                 """'save_targets_ensemble'")""",
+                """AttributeError("'BackendMock' object has no attribute """
```

Comment:
I cannot spot the difference, which one is it?

Reply:
what do you mean? So I realised that we were never setting up the logger in the backend and therefore I added a line [here](https://github.com/ravinkohli/Auto-PyTorch/blob/36ae93c6a8bf83c8ecca8601946e473213a184b4/autoPyTorch/evaluation/abstract_evaluator.py#L334) and also needed to add this to the test as we are not mocking this function